ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-stable-4.15.yaml

```diff
@@ -443,6 +443,7 @@ tests:
     cluster_profile: equinix-ocp-metal-qe
     env:
       AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      PRIMARY_NET: ipv4
```

Comment:
Perhaps we can set `PRIMARY_NET: ipv4` in workflow `baremetal-lab-upi-dual-stack` ?
That way, we only need to change one place.